src/views/registration-js/attendees/fields/birth.php

```diff
@@ -28,7 +29,7 @@
 	<div <?php tribe_classes( $classes ); ?> >
 		<label
 			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
-			for="<?php echo esc_attr( $birth_field->month_id ); ?>"
+			for="<?php echo esc_attr( $field_name ); ?>"
```

Comment:
The value here should be `$option_id`

Reply:
https://www.w3schools.com/tags/att_label_for.asp

Reply:
Nice catch man, thanks! https://github.com/moderntribe/event-tickets/pull/1676/commits/5456e6aefbb3fb43f7b744febebdab0eac9310b6